### PR TITLE
add healthcheck http handler

### PR DESCRIPTION
**Pull request by RaphaelVRossi**
Create http healthcheck endpoint for Remotecv worker.

We are running Remotecv process inside a minimal docker image, and we need to check if the process is still alive

### CODE REVIEW

File: remotecv/worker.py [hunk 1]
```diff
@@ -84,6 +97,8 @@ def main(params=None):
     )
 
     other_group = parser.add_argument_group("Other arguments")
+    other_group.add_argument("--server_port", default=8080, type=int, help="Server http port")
+    other_group.add_argument("--with_healthcheck", default=False, type=bool, action=argparse.BooleanOptionalAction, help="Start an healthchecker http endpoint")
```
Comment (marcelometal) [inline remotecv/worker.py:101]: Replace `with_healthcheck` with `with-healthcheck`

File: remotecv/worker.py [hunk 2]
```diff
@@ -84,6 +97,8 @@ def main(params=None):
     )
 
     other_group = parser.add_argument_group("Other arguments")
+    other_group.add_argument("--server_port", default=8080, type=int, help="Server http port")
```
Comment (marcelometal) [inline remotecv/worker.py:100]: Replace `server_port` with `server-port`

File: remotecv/worker.py (hunk 1 above)
Comment (marcelometal) [reply remotecv/worker.py:101]: Wrap this long line ;)

Comment (marcelometal) [reply remotecv/worker.py:101]: Please, add a note in README about how to use this new arg

File: setup.py [hunk 3]
```diff
@@ -12,6 +12,7 @@
     "pytest==7.*,>=7.0.0",
     "pytest-cov==3.*,>=3.0.0",
     "pytest-asyncio==0.*,>=0.18.0",
+    "requests==2.*,>=2.27.1",
```
Comment (scorphus) [inline setup.py:15]:
```suggestion
```

File: tests/test_healthcheck_handler.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from unittest import TestCase
+
+import requests
```
Comment (scorphus) [inline tests/test_healthcheck_handler.py:6]:
```suggestion
from urllib import request
```

File: tests/test_healthcheck_handler.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+from remotecv import worker
+from remotecv.utils import config
+
+
+class HealthCheckTestCase(TestCase):
+    def test_should_be_200_and_working(self):
+        config.server_port = 8888
+
+        worker.start_http_server()
+        response = requests.get(f"http://localhost:{config.server_port}")
```
Comment (scorphus) [inline tests/test_healthcheck_handler.py:18]:
```suggestion
        response = request.urlopen(f"http://localhost:{config.server_port}")
```

File: tests/test_healthcheck_handler.py [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+        expect(response.status_code).to_equal(200)
+        expect(response.text).to_equal("WORKING")
```
Comment (scorphus) [inline tests/test_healthcheck_handler.py:21]:
```suggestion
        expect(response.status).to_equal(200)
        expect(response.read().encode()).to_equal("WORKING")
```

**Review by scorphus — CHANGES_REQUESTED** [review]
The Python Standard Library serves so many purposes. Let's use it! We don't need yet another dependency. 🙂

File: setup.py (hunk 3 above)
Comment (RaphaelVRossi) [reply setup.py]: Done!

File: tests/test_healthcheck_handler.py (hunk 5 above)
Comment (RaphaelVRossi) [reply tests/test_healthcheck_handler.py]: Done!

File: tests/test_healthcheck_handler.py (hunk 4 above)
Comment (RaphaelVRossi) [reply tests/test_healthcheck_handler.py]: Done!

File: tests/test_healthcheck_handler.py (hunk 6 above)
Comment (RaphaelVRossi) [reply tests/test_healthcheck_handler.py]: Done!

File: README.md [hunk 7]
```diff
@@ -45,6 +45,20 @@ Run:
 make run
 ```
 
+After installing the project, you can execute
+
+```sh
+remotecv
+```
+
+If you want a healthcheck handler, you must pass a argument in remotecv
```
Comment (scorphus) [inline README.md:54]: Missed that:
```suggestion
If you want a healthcheck handler, you must pass an argument in remotecv
```

File: README.md [hunk 8]
```diff
@@ -45,6 +45,20 @@ Run:
...
+execution. You can also specify the http server port, the default http server
+port is 8080
```
Comment (scorphus) [inline README.md:56]:
```suggestion
port is 8080.
```
\- "Pô, Pablito. Mas tu é nitty picky, hein?"
\- 👀

**Review by scorphus — APPROVED** [review]

File: README.md (hunk 8 above)
Comment (RaphaelVRossi) [reply README.md]: El Pablito não deixa passar uma.

**Pull request merged**